File: benchmarks/json-parse-hoist/baseline/workload.py
from solution import score
import json
records = [{"value": i % 17} for i in range(30000)]
config = json.dumps({"multiplier": 7, "name": "benchmark", "flags": list(range(20))})
for _ in range(8):
    result = score(records, config)
assert result > 0

File: benchmarks/js-map-index/baseline/workload.js
const { resolve } = require("./solution");
const records = Array.from({ length: 7000 }, (_, i) => ({ id: i, value: i * 3 }));
const ids = Array.from({ length: 7000 }, (_, i) => 6999 - i);
for (let i = 0; i < 3; i++) {
  const result = resolve(ids, records);
  if (result.length !== 7000) throw new Error("bad result");
}
